datalad/support/annexrepo.py

```diff
@@ -3430,7 +3430,7 @@ def annexstatus(self, paths=None, untracked='all'):
         """
         .. deprecated:: 0.16
             Use get_content_annexinfo() or the test helper
-            datalad/utils/get_annexstatus() instead.
+            datalad/tests/utils/get_annexstatus() instead.
```

Comment:
oh, nice catch!  For all sphinx experts, should we fix that into

```suggestion
            :py:func:`datalad.tests.utils.get_annexstatus` instead.
```

or not worth the CI CPU cycles? ;)